modules/API/API.js

```diff
@@ -166,6 +169,26 @@ function initCommands() {
             } catch (err) {
                 logger.error('Failed sending endpoint text message', err);
             }
+        },
+        'start-recording': () => {
+            const { conference } = APP.store.getState()['features/base/conference'];
+
+            conference && conference.startRecording({
+                mode: JitsiRecordingConstants.mode.FILE,
+                appData: JSON.stringify({
+                    'file_recording_metadata': {
+                        'share': false
+                    }
+                })
+            });
+        },
+        'stop-recording': () => {
+            const { conference } = APP.store.getState()['features/base/conference'];
```

Comment:
get the state on an auxiliary var please, instead of fetching it twice

Reply:
done